wasm/tools/fixtures/runner.py

```diff
@@ -72,19 +74,16 @@ def instantiate_module_from_wasm_file(
                 raise Unlinkable(f"Unlinkable module: {import_['module']}")
 
             sub_module = registered_modules[import_["module"]]
-            externval = None
+
             for export in sub_module["exports"]:
                 if export["name"] == import_["name"]:
                     externval = export["value"]
-            if externval is None:
+                    break
+            else:
                 raise Unlinkable("Unlinkable module: export name not found")
-            if externval[0] != import_["desc"][0]:
-                raise Unlinkable("Unlinkable module: TODO figure out what this means")
```

Comment:
Deleting `TODO figure out what this means` when I don't know what it means scares me.

Reply:
Basically this code path wasn't being used because this validation now happens internally with the `Unlinkable` exception originating from inside the module, and I'm reasonably confident that it now has a friendly/useful error message.